src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -34,6 +57,29 @@ def validate_datetime(string):
     date_format = '%Y-%m-%dT%H:%MZ'
     return datetime.strptime(string, date_format)
 
+def validate_entity(namespace):
+    ''' Converts a list of key value pairs into a dictionary. Ensures that required
+    RowKey and PartitionKey are converted to the correct case and included. '''
+    values = dict(x.split('=', 1) for x in namespace.entity)
+    keys = values.keys()
+    for key in keys:
+        if key.lower() == 'rowkey':
+            val = values[key]
+            del values[key]
+            values['RowKey'] = val
+        elif key.lower() == 'partitionkey':
+            val = values[key]
+            del values[key]
+            values['PartitionKey'] = val
+    keys = values.keys()
+    missing_keys = 'RowKey ' if 'RowKey' not in keys else ''
+    missing_keys = '{}PartitionKey'.format(missing_keys) \
```

Comment:
>{}PartitionKey [](start = 20, length = 14)

Do you need a space after the {}?
Otherwise it looks like the ArgumentError below could print RowKeyPartitionKey.
Is that intended?